sdks/rust/golem-rust/Cargo.toml

```diff
@@ -1,31 +1,41 @@
 [package]
-name = "golem-rust"
-version = "0.0.0"
+description = "Golem Rust tooling library that facilitates writing Golem backends in Rust"
 edition = "2021"
-license = "Apache-2.0"
 homepage = "https://golem.cloud"
+license = "Apache-2.0"
+name = "golem-rust"
 repository = "https://github.com/golemcloud/golem-rust"
-description = "Golem Rust tooling library that facilitates writing Golem backends in Rust"
+version = "0.0.0"
 
 [lib]
-path = "src/lib.rs"
 crate-type = ["rlib"]
+path = "src/lib.rs"
 
 [dependencies]
-golem-rust-macro = { path = "../golem-rust-macro", version = "0.0.0", optional = true }
-serde = { version = "1", optional = true }
-serde_json = { version = "1", optional = true }
-uuid = { version = "1", features = ["v4"] }
-golem-wasm-rpc = { version = "1.3.0-rc9pre", default-features = false, features = [
-    "stub",
-] }
-wit-bindgen = { version = "=0.40.0" }
+golem-rust-macro = {path = "../golem-rust-macro", version = "0.0.0", optional = true}
+
+# TODO; This needs to be updated to golem-wasm
```

Comment:
Have to start publishing new `dev` releases but I don't know what version to use :)